sdk/appconfiguration/azure-data-appconfiguration/README.md

```diff
@@ -37,7 +37,7 @@ AppConfiguration to use Netty HTTP client.
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-core-http-netty</artifactId>
-    <version>1.0.0</version>
+    <version>1.2.0</version>
```

Comment:
Should this be marked as a dependency version instead of current version?

Reply:
agreed. This should be a dependency version

Reply:
Spoke to Alan and agreed to keep it current because once we release that is what people will see in this version of README.